uid/snowid_test.go

```diff
@@ -46,12 +47,15 @@ func TestNode_Generate_ConcurrencyAndUniqueness(t *testing.T) {
 	}
 
 	// Pick a random set of unique node IDs each time
-	nodeMax := -1 ^ (-1 << nodeBits)
+	nodeMax := -1 ^ (-1 << nodeBits) // modeMax=1024
 	nodeIDs := make([]int64, nodeMax)
 	for n := 0; n < nodeMax; n++ {
 		nodeIDs[n] = int64(n)
 	}
-	rand.Shuffle(nodeMax, func(i, j int) {
+	seed := time.Now().UnixNano()
+	t.Log("seed", seed)
+	r := rand.New(rand.NewSource(seed))
```

Comment:
Use a different seed each time so that we actually get a random shuffle. Without this it is likely the random would always produce the same results because it gets seeded to the same value (`1`).

Also print the seed to the test log. If this test fails, that gives us an easy way to reproduce (by  hardcoding that seed value).